### PR TITLE
Fixing type lookup

### PR DESCRIPTION
**Pull request by josenavas**
There was an error in the way the error was checked. This prevents us to actually see the source of the error. It will be great if we can have this merged fast, as it is blocking work on the live system.

### CODE REVIEW

**Comment by antgonza** [comment]
:+1:

File: qiita_db/metadata_template/util.py [hunk 1]
```diff
@@ -25,6 +25,20 @@
     from string import letters, digits
 
 
+def type_lookup(dtype):
+    """Lookup function """
```
Comment (ElDeveloper) [inline qiita_db/metadata_template/util.py]: Can you make this numpydoc compliant?

**Comment by ElDeveloper** [comment]
Looks good, 1 comment though.

File: qiita_db/metadata_template/util.py (hunk 1 above)
Comment (josenavas) [inline qiita_db/metadata_template/util.py]: hahaha sure! I think I haven't finished the docstring >.< too involved on the other issue...

**Comment by josenavas** [comment]
Thanks @ElDeveloper 
I addressed the comment!

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**